include/galsim/hsm/PSFCorr.h

```diff
@@ -122,6 +122,10 @@ namespace hsm {
      *                           KSB method of PSF correction.  Warning: deviating from default
      *                           value of 2 results in code running more slowly, and results have
      *                           not been significantly tested.
+     * @param epsilon            Accuracy (in x0, y0, and sigma as a fraction of sigma) when
```

Comment:
I think you're right that it should be that, but I'm not sure if it actually is.  I tried to follow the code in hsm, but got a little lost, so I think I'll punt and let Rachel weigh in on whether this text should be changed.  (I just copied it from a comment in the C++ code.)  

I.e. is the x0,y0 convergence relative to sigma, or it units of pixels?